app/src/main/java/org/dslul/openboard/inputmethod/latin/spellcheck/AndroidWordLevelSpellCheckerSession.java

```diff
@@ -62,6 +64,15 @@ public abstract class AndroidWordLevelSpellCheckerSession extends Session {
     private static final String quotesRegexp =
             "(\\u0022|\\u0027|\\u0060|\\u00B4|\\u2018|\\u2018|\\u201C|\\u201D)";
 
+    private static final Map<Integer, String> scriptToPunctuationRegexMap = new TreeMap<>();
+
+    static {
+        scriptToPunctuationRegexMap.put(
```

Comment:
There needs to be a way to ignore language specific punctuation in addition to the standard English ones. See the usage below.